### PR TITLE
add name arg to direction and distance

### PR DESCRIPTION
**Pull request by raybellwaves**
Follow on from https://github.com/anitagraser/movingpandas/pull/151

Add name arg to `add_direction` and `add distance`.

### CODE REVIEW

**Review by anitagraser — APPROVED** [review]

File: movingpandas/trajectory.py [hunk 1]
```diff
@@ -772,12 +789,14 @@ def add_distance(self, overwrite=False):
         overwrite : bool
             Whether to overwrite existing distance values (default: False)
         """
-        if DISTANCE_COL_NAME in self.df.columns and not overwrite:
+        self.distance_col_name = name
+        if self.distance_col_name in self.df.columns and not overwrite:
```
Comment (anitagraser) [inline movingpandas/trajectory.py:793]: Could you add another test for this, so we can bring the test coverage up?

Comment (raybellwaves) [reply movingpandas/trajectory.py:793]: Would what the test be for exactly?

AFAICT all distance tests match speed tests for this functionality?

`test_add_distance_with_name`
`test_add_distance_doesnt_change_existing_distance`

Comment (raybellwaves) [reply movingpandas/trajectory.py:793]: Am I right the PR coverage diff:
https://codecov.io/gh/anitagraser/movingpandas/compare/0e412b59bfd1234bd6bb4345c0c316f5770f1950...48c610b282549c5f1663878374d8a37fe27646b7/diff
is coming from 
`if hasattr(self, "distance_col_name")`...

Comment (anitagraser) [reply movingpandas/trajectory.py:793]: You are right. That's the coverage diff.

**Pull request merged**